src/components/loginform/helperFunction/regexValidation.js

```diff
@@ -0,0 +1,27 @@
+function regexValidation(user) {
```

Comment:
@Sheyitofunmi Everything looks good so far; I would like to merge @TaylorDJones11's Sign-Up-Firebase-Connect [PR](https://github.com/AritDeveloperCircle/findyourpeople/pull/88) first so that you have access to the `regexValidation.js` helper function file (and prevent duplication).